main.js

```diff
@@ -0,0 +1,138 @@
+var form = document.getElementsByClassName('post-form')[0];
+var i = 0
+
+form.addEventListener('submit', function(event1) {
```

Comment:
Why not use jQuery here? Also, why is everything nested under this event handler?